[pbc] Build failure

Package: pbc:x64-windows -> 0.5.14#8

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2023-08-09-9990a4c9026811a312cb2af78bf77f3d9d288416
    vcpkg-scripts version: 2c401863d 2023-09-13 (5 hours ago)

**To Reproduce**

`vcpkg install pbc`
**Failure logs**

```
-- Note: pbc only supports static library linkage. Building static library.
-- Using cached blynn-pbc-fbf4589036ce4f662e2d06905862c9e816cf9d08.tar.gz.
-- Extracting source C:/Utils/vcpkg/downloads/blynn-pbc-fbf4589036ce4f662e2d06905862c9e816cf9d08.tar.gz
-- Applying patch windows.patch
-- Using source at C:/Utils/vcpkg/buildtrees/pbc/src/e816cf9d08-73bced3986.clean
-- Found external ninja('1.11.0').
-- Getting CMake variables for x64-windows
-- Building e816cf9d08-73bced3986.clean/pbcwin/projects/pbclib.vcxproj for Release
-- Building e816cf9d08-73bced3986.clean/pbcwin/projects/pbclib.vcxproj for Debug
CMake Error at scripts/cmake/vcpkg_install_copyright.cmake:21 (file):
  file INSTALL cannot find "C:/Utils/vcpkg/
  C:/Utils/vcpkg/buildtrees/pbc/src/e816cf9d08-73bced3986.clean/COPYING":
  File exists.
Call Stack (most recent call first):
  ports/pbc/portfile.cmake:95 (vcpkg_install_copyright)
  scripts/ports.cmake:147 (include)



```

--- a/ports/pbc/portfile.cmake
+++ b/ports/pbc/portfile.cmake
@@ -47,17 +47,17 @@ else()
         PATCHES windows.patch
     )
 
-    find_path(MPIR_INCLUDE_DIR "gmp.h" HINTS ${CURRENT_INSTALLED_DIR} PATH_SUFFIXES include)
+    find_path(MPIR_INCLUDE_DIR "gmp.h" HINTS "${CURRENT_INSTALLED_DIR}" PATH_SUFFIXES include)
     if(NOT MPIR_INCLUDE_DIR)
         message(FATAL_ERROR "GMP includes not found")
     endif()
 
-    find_library(MPIR_LIBRARIES_REL NAMES "mpir.lib" HINTS ${CURRENT_INSTALLED_DIR} PATH_SUFFIXES lib)
+    find_library(MPIR_LIBRARIES_REL NAMES "mpir.lib" HINTS "${CURRENT_INSTALLED_DIR}" PATH_SUFFIXES lib)
     if(NOT MPIR_LIBRARIES_REL)
         message(FATAL_ERROR "mpir library not found")
     endif()
 
-    find_library(MPIR_LIBRARIES_DBG NAMES "mpir.lib" HINTS ${CURRENT_INSTALLED_DIR} PATH_SUFFIXES debug/lib)
+    find_library(MPIR_LIBRARIES_DBG NAMES "mpir.lib" HINTS "${CURRENT_INSTALLED_DIR}" PATH_SUFFIXES debug/lib)
     if(NOT MPIR_LIBRARIES_DBG)
         message(FATAL_ERROR "mpir debug library not found")
     endif()
@@ -77,22 +77,22 @@ else()
     endif()
 
     # PBC expects mpir directory in build root
-    get_filename_component(SOURCE_PATH_PARENT ${SOURCE_PATH} DIRECTORY)
-    file(REMOVE_RECURSE ${SOURCE_PATH_PARENT}/mpir)
-    file(MAKE_DIRECTORY ${SOURCE_PATH_PARENT}/mpir)
+    get_filename_component(SOURCE_PATH_PARENT "${SOURCE_PATH}" DIRECTORY)
+    file(REMOVE_RECURSE "${SOURCE_PATH_PARENT}/mpir")
+    file(MAKE_DIRECTORY "${SOURCE_PATH_PARENT}/mpir")
     file(GLOB FILES ${MPIR_INCLUDE_DIR}/gmp*.h)
     file(COPY ${FILES} ${MPIR_LIBRARIES_REL} DESTINATION "${SOURCE_PATH_PARENT}/mpir/${LibrarySuffix}/${Platform}/Release")
     file(COPY ${FILES} ${MPIR_LIBRARIES_DBG} DESTINATION "${SOURCE_PATH_PARENT}/mpir/${LibrarySuffix}/${Platform}/Debug")
 
-    get_filename_component(SOURCE_PATH_SUFFIX ${SOURCE_PATH} NAME)
-    vcpkg_msbuild_install(SOURCE_PATH ${SOURCE_PATH_PARENT}
-        PROJECT_SUBPATH ${SOURCE_PATH_SUFFIX}/pbcwin/projects/pbclib.vcxproj
+    get_filename_component(SOURCE_PATH_SUFFIX "${SOURCE_PATH}" NAME)
+    vcpkg_msbuild_install(SOURCE_PATH "${SOURCE_PATH_PARENT}"
+        PROJECT_SUBPATH "${SOURCE_PATH_SUFFIX}/pbcwin/projects/pbclib.vcxproj"
         RELEASE_CONFIGURATION "Release${ConfigurationSuffix}"
         DEBUG_CONFIGURATION "Debug${ConfigurationSuffix}"
         OPTIONS /p:SolutionDir=../
     )
 
-    vcpkg_install_copyright(FILE_LIST " ${SOURCE_PATH}/COPYING")
+    vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
     file(COPY "${SOURCE_PATH}/include/" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
     # clean up mpir stuff
     file(REMOVE "${CURRENT_PACKAGES_DIR}/lib/mpir.lib" "${CURRENT_PACKAGES_DIR}/debug/lib/mpir.lib")

--- a/ports/pbc/vcpkg.json
+++ b/ports/pbc/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "pbc",
   "version": "0.5.14",
-  "port-version": 8,
+  "port-version": 9,
   "description": "Pairing-Based Crypto library provides low-level routines for pairing-based cryptosystems.",
   "homepage": "https://crypto.stanford.edu/pbc",
   "license": "LGPL-3.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6286,7 +6286,7 @@
     },
     "pbc": {
       "baseline": "0.5.14",
-      "port-version": 8
+      "port-version": 9
     },
     "pcapplusplus": {
       "baseline": "22.11",

--- a/versions/p-/pbc.json
+++ b/versions/p-/pbc.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6789ed942578afe4baac765d5e7549d421c25c76",
+      "version": "0.5.14",
+      "port-version": 9
+    },
+    {
       "git-tree": "096b1e1d4077f9fbe44e8b0a6fa306e83270c632",
       "version": "0.5.14",
       "port-version": 8

